libxcomposite will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxcomposite it will fail in the following way:

```
2024/11/12 23:12:36 INFO running step "test/pkgconf"
2024/11/12 23:12:36 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:36 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:36 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:36 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:36 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + basename /home/build/melange-out/libxcomposite-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + dev_pkg=libxcomposite-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + apk info -L libxcomposite-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + grep -q ^Name: usr/lib/pkgconfig/xcomposite.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + basename usr/lib/pkgconfig/xcomposite.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + lib_name=xcomposite uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + echo usr/lib/pkgconfig/xcomposite.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + grep -q '^usr/lib/pkgconfig/xcomposite.pc$\|^usr/share/pkgconfig/xcomposite.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + pkgconf --exists xcomposite uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + grep -q ^Version: usr/lib/pkgconfig/xcomposite.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + pkgconf --modversion xcomposite uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 INFO 0.4.6 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + grep -q ^Libs: usr/lib/pkgconfig/xcomposite.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + pkgconf --libs xcomposite uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 INFO -lXcomposite uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + grep -q ^Cflags: usr/lib/pkgconfig/xcomposite.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN + pkgconf --cflags xcomposite uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN Package x11 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN Perhaps you should add the directory containing `x11.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN Package 'x11', required by 'xcomposite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 WARN Package 'xfixes', required by 'xcomposite', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:36 INFO pod f05e1e286d599f9066661e06eb199726b8ef9e1c6c914ce7116940163f665629 terminated
2024/11/12 23:12:36 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:36 INFO   workspace dir: /home/user/tmp/melange-workspace-73216462
2024/11/12 23:12:36 INFO   guest dir: /home/user/tmp/melange-guest-3433865444
2024/11/12 23:12:36 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxcomposite] Error 1
```

--- a/libxcomposite.yaml
+++ b/libxcomposite.yaml
@@ -45,8 +45,13 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - libx11-dev
         - libxext-dev
+        - libxfixes-dev
     description: libxcomposite dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxcomposite-doc
     pipeline:

